racingcar-java/src/test/java/StringTest.java

```diff
@@ -0,0 +1,45 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @DisplayName("요구사항 1-1")
+    @Test
+    void splitTest() {
+        String[] result = "1,2".split(",");
+        assertThat(result).contains("2", "1");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @DisplayName("요구사항 1-2")
+    @Test
+    void splitTest2() {
+        String[] result = "1".split(",");
+        assertThat(result).containsExactly("1");
+    }
+
+    @DisplayName("요구사항 2")
+    @Test
+    void substringTest() {
+        String data = "(1,2)";
+        int left = data.indexOf("(");
+        int right = data.indexOf(")");
+        String result = data.substring(left + 1, right);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @DisplayName("요구사항 3")
+    @Test
+    void charAtTest() {
+        String data = "abc";
+        assertThatThrownBy(() -> {
+            data.charAt(5);
+        }).isInstanceOf(StringIndexOutOfBoundsException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> data.charAt(5))
            .isInstanceOf(StringIndexOutOfBoundsException.class);
```
람다식을 사용할 때 void 메소드의 라인이 추가되지 않는 경우에는 {}를 사용하지 않아도 됩니다.